app/components/ui/left-menu-collapsable-container/component.js

```diff
@@ -0,0 +1,45 @@
+import Ember from "ember";
+import computed from "ember-computed-decorators";
+
+export default Ember.Component.extend({
+  preferencesService: Ember.inject.service(),
+  classNames: ["col"],
+
+  isSmallExpanded: Ember.computed.alias("preferencesService.preferencesData.isSmallExpanded"),
```

Comment:
I think we should remove one of these flags, and rename the remaining one to: "ordersNavOpen". Then we should just rely on the css rule to show open or close state depending on if this flag is true + if a media query breakpoint has been hit.

We should look at using `classNameBindings` to then bind the desired class.

I even think it will be best to use what you already have with `collapsable-container` Then rest can be handled in css with media queries.

If you abstract the collapse trigger to be something you can specify, then we can just reuse that component and get rid of this component.

As a general rule, the prefs settings keys should be descriptive enough so that we could get an idea of what they reference just by reading the key. `isSmallExpanded` doesn't really give us this info.